### DISCUSSION TITLE
Possible to have a page within Quarto Dashboard without cards, formatted as markdown?

### DISCUSSION THREAD

**Discussion by jburos** [quarto-dev/quarto-cli#9066] [Q&A]
### Description

I love the recently released quarto dashboard format, however I have a question.

I often have a dashboard where I want to include say 2 pages of results, and 1 page with technical details (say, a model description). 

My question is about supporting a scenario where that 3rd page contains markdown with level 2 and 3 headings, which I might want to navigate with a TOC. Right now, these get converted to "cards" which is not ideal for a technical (mostly text) markdown document. 

As an example, consider this vetiver dashboard: https://www.isabelizimm.me/vetiverdashboard/#

I have a version of this but the model description is more involved, and I would like to separate it using section headings (level 2, level 3, etc).

````qmd
---
title: "Exposure-Response Analysis"
format:
  dashboard:
    orientation: columns
---

```{r}
#| label: setup
library(gtsummary)
library(tidyverse)
library(tidybayes)
```

# Observed Data

```{r}
#| title: Subjects Data
table1 <- 
  trial |>
  tbl_summary(include = c(age, grade, response))
```

# Model Description

The model used here is a flexible formulation of the Stein-Fojo model for change over time, in which terms can be added or modified using a formula syntax. In the sections below, we introduce the framework before discussing particular parameterizations investigated for this analysis.

## Observation model

The likelihood of observation $i$, independently over all $i$, is
$$
\log(y_i + \delta) \sim \mathcal{N}(f_i, \sigma^2),
$$
where

$$
f_i = f(\mathbf{x}_i)= \sum_{j=1}^J f^{(j)}(\mathbf{x}_i)
$$
is the expected log PSA, $\sigma$ is an unknown parameter, and 
the $\delta$ value is a constant. We use $y_i$ to denote the observed PSA 
(prostate-specific antigen), and $\mathbf{x}_i$ the corresponding covariate
vector. In the analyses presented here,
$\mathbf{x}_i = \{t_i, \text{id}_i, \text{arm}_i\}$ where $t_i$ is time from
the start of treatment, $\text{id}_i$ is the subject identifier (integer),
and $\text{arm}_i$ is the treatment arm or cohort. The functions 
$f^{(j)}$, $j=1, \ldots, J$ are the additive model terms
and they are detailed next.

### Model terms

blah blah blah... 

# Parameters

## Population Parameters

```{r}
#| title: Summarize Pars
set.seed(5)
n = 10
n_condition = 5
ABC =
  tibble(
    condition = rep("A","B","C","D","E"), n),
    response = rnorm(n * 5, c(0,1,2,1,-1), 0.5)
  )
 ABC |>
  ggplot(aes(y = condition, x = response)) +
  geom_point()
```
````

note: examples taken from [tidybayes](https://mjskay.github.io/tidybayes/articles/tidy-brms.html#setup) and [gtsummary](https://www.danieldsjoberg.com/gtsummary/) doc.

**Comment by mcanouil** [comment]
You can't have a TOC as it is a dashboard and dashboards don't have TOC.

I suggest you integrate your dashboard in a website (it should work) rather than trying to do the other way around.

**Comment by jburos** [comment]
> You can't have a TOC as it is a dashboard and dashboards don't have TOC.

Makes sense. Is it possible to keep headers formatted as headers instead of making them into rows/columns or cards & boxes?

> I suggest you integrate your dashboard in a website (it should work) rather than trying to do the other way around.

Thank you for the suggestion!  I haven't used website format within the quarto-verse. I will try to create an example.

**Reply by mcanouil** [reply]
> Makes sense. Is it possible to keep headers formatted as headers instead of making them into rows/columns or cards & boxes? 

Not at the "top-level".
